frontend/package.json

```diff
@@ -7,6 +7,9 @@
     "name": "John Doe",
     "company": "ACME, Inc."
   },
+  "scripts": {
```

Comment:
I'm not sure how that works, actually. For now let's not include this change, since it's not directly related to the rest of the patch.